src/onegov/org/views/resource.py

```diff
@@ -595,20 +596,21 @@ def view_resource(
 def handle_delete_resource(self: Resource, request: 'OrgRequest') -> None:
 
     request.assert_valid_csrf_token()
-
-    # FIXME: should we move this attribute to the base class?
-    #        it could just be statically True or False by default
-    if not self.deletable:  # type:ignore[attr-defined]
```

Comment:
I think this condition should stay, didn't we discuss, that resources with future reservation should still remain undeletable? In that case you need to change the `deletable` property, rather than remove this condition.